bucket/sysinternals.json

```diff
@@ -59,7 +59,11 @@
                 "ctrl2cap.exe",
                 "Dbgview.exe",
                 "Desktops.exe",
-                "disk2vhd.exe",
+                [
+                    "disk2vhd64.exe",
+                    "disk2vhd"
+                ],
+                "disk2vhd64.exe",
```

Comment:
```suggestion

```

This is not required then

Reply:
It is an alias to make sure that if the user calls disk2vhd the 64bit ver will be called. I can see the other program in this suite do the same thing.